tools/update-snapcraft

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+# Authors: Mert Kirpici <mert.kirpici@canonical.com>
+# Copyright (C) 2023 Canonical
```

Comment:
the snap as a whole is licensed under the Apache v2.0 license. Would you explicitly want this script to *not* be under that license?